spec/controllers/subscriptions_controller_spec.rb

```diff
@@ -101,4 +101,67 @@
       end
     end
   end
+
+  describe "#update" do
```

Comment:
is this test necessary, given the system spec? have I tested correctly?

Reply:
Are you testing anything more here than you are in the other test? If so, perhaps, if not, no.